compilers/concrete-compiler/compiler/lib/Dialect/FHE/Analysis/MANP.cpp

```diff
@@ -1217,7 +1215,7 @@ llvm::APInt RoundEintOp::sqMANP(llvm::APInt a) {
 }
 } // namespace FHE
 
-namespace FHELinalg {
+namespace FHELinalg { /////// !manp
```

Comment:
??? not clear.
TODO: add a sub/enclosing namespace "manp" ?

Reply:
No idea, what this is 🤣 Removed !